### ISSUE TITLE
[Bug]: Vampire lords T-posing.

### ISSUE THREAD

**Issue by WylerL** [Monitor221hz/Pandora-Behaviour-Engine-Plus#306]
### Check for duplicate issues

- [X] I have verified that there are no duplicate [issues](https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/issues).

### Version

v2.3.0-beta

### Provide Instructions on Reproducing the Problem

I'm aware this isn't really a Pandora issue, and is more issue with my mods but I'm at my wits end and am really just looking for any advice I can try to fix this.

I had some creature animation mods that effected Vampire Lords, mostly Sexlab stuff. I removed them some time ago, and have ran the engine multiple times since then, and cleared my cache. I no longer have any mods that should be effecting this entity at all, and I did have other t-pose issues after getting rid of Sexlab but I seem to have fixed everything except for Vampire Lords in particular. 

I just really want to know if anyone has any solutions for this, I'm stumped and there seems to very little information online about these t-posing issues that isn't just "Run FNIS/Nemesis.".

### Provide Relevant Files

INFO : Skyrim Patcher 2.0.4-alpha
ERROR : Nemesis Assembler > File E:\SteamLibrary\steamapps\common\Skyrim Special Edition\data\Nemesis_Engine\mod\colisc\draugrbehavior\#0115.txt > Load > FAILED > The 'hkobject' start tag on line 1 position 4 does not match the end tag of 'hkparam'. Line 11, position 5.
ERROR : Nemesis Assembler > File E:\SteamLibrary\steamapps\common\Skyrim Special Edition\data\Nemesis_Engine\mod\colisc\draugrbehavior\#0119.txt > Load > FAILED > The 'hkobject' start tag on line 1 position 4 does not match the end tag of 'hkparam'. Line 11, position 5.
WARN : Dispatcher > "Pandora Base" > dragonproject~dragonbehavior > Replace > Element > #0885/legs/Element0/maxAnkleHeightMS > FAILED
WARN : Dispatcher > "Pandora Base" > dragonproject~dragonbehavior > Replace > Element > #0885/legs/Element0/hipIndex > FAILED
WARN : Dispatcher > "Pandora Base" > dragonproject~dragonbehavior > Replace > Element > #0885/legs/Element0/kneeIndex > FAILED
WARN : Dispatcher > "Pandora Base" > dragonproject~dragonbehavior > Replace > Element > #0885/legs/Element0/ankleIndex > FAILED
WARN : Dispatcher > "Pandora Base" > dragonproject~dragonbehavior > Replace > Element > #0885/legs/Element0/isPlantedMS > FAILED
WARN : Validator > defaultmale~0_master > Duplicate Event > OST_ExitAnim > Index > 1775 > SKIPPED
WARN : Validator > defaultmale~0_master > Duplicate Event > OST_ExitAnim > Index > 1790 > SKIPPED
WARN : Validator > defaultmale~0_master > Duplicate Event > OST_ExitAnim > Index > 2169 > SKIPPED
WARN : Validator > defaultmale~0_master > Duplicate Event > OST_ExitAnim > Index > 2194 > SKIPPED
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > Version V7.2 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAprefix xpe > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _1hmeqp 5 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _2hmeqp 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _2hweqp 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _axeeqp 5 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _bowatk 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _boweqp 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _bowidle 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _maceqp 2 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _dageqp 6 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _magatk 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _magcastmt 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _magcon 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _magidle 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _magmt 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _sprint 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > AAset _shout 1 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > T xpe3_Dag_Unequip.hkx Unequip_OutMoving -0.333 Unequip_Out -0.00 BeginWeaponSheathe 0.00 > File > FNIS_XPMSE_List.txt
WARN : FNIS Animlist > New Animation > From Line > FAILED > Line > T xpe4_Dag_Unequip.hkx Unequip_OutMoving -0.333 Unequip_Out -0.00 BeginWeaponSheathe 0.00 > File > FNIS_XPMSE_List.txt
INFO : Engine launched with configuration: Skyrim SE/AE. Do not exit before the launch is finished.
INFO : Pandora Mod 1 : Maxsu Block Overhaul - v.1.0.0
INFO : Pandora Mod 2 : Precision - v.1.0.0
INFO : Pandora Mod 3 : Precision Creatures - v.1.0.0
INFO : Pandora Mod 4 : Eating Animations and Sounds SE - v.1.0.0
INFO : Pandora Mod 5 : Payload Interpreter - v.1.0.0
INFO : Pandora Mod 6 : GP Offset Movement Animation - v.1.0.0
INFO : Pandora Mod 7 : Ultimate Potion Animation NG - v.1.0.0
INFO : Pandora Mod 8 : Lovemaking Compendium - v.1.0.0
INFO : Pandora Mod 9 : Night Blooming Violets - v.1.0.0
INFO : Pandora Mod 10 : MNBVwa - v.1.0.0
INFO : Pandora Mod 11 : OAnims - v.1.0.0
INFO : Pandora Mod 12 : OpenSex - v.1.0.0
INFO : Pandora Mod 13 : OStim - v.1.0.0
INFO : Pandora Mod 14 : True Directional Movement - 360 Horse Archery - v.1.0.0
INFO : Pandora Mod 15 : True Directional Movement - Procedural Leaning - v.1.0.0
INFO : Pandora Mod 16 : True Directional Movement - Headtracking - v.1.0.0
INFO : Pandora Mod 17 : Animation Teleport Bug Fix - v.1.0.0
INFO : Pandora Mod 18 : Pandora Base - v.1.0.0
INFO : FNIS Mod 1 : FNIS_openani_interaction_List
INFO : FNIS Mod 2 : FNIS_openani_redress_List
INFO : FNIS Mod 3 : FNIS_openani_solo_List
INFO : FNIS Mod 4 : FNIS_ImmersiveInteractions_List
INFO : FNIS Mod 5 : _FNIS_openani_nemesis_List
INFO : FNIS Mod 6 : FNIS_OCR_List
INFO : FNIS Mod 7 : FNIS_IEA_List
INFO : FNIS Mod 8 : FNIS_openani_furniture_List
INFO : FNIS Mod 9 : ATT_openani_animlist
INFO : FNIS Mod 10 : FNIS_openani_main_List
INFO : FNIS Mod 11 : ATT_MNBVwa_animlist
INFO : FNIS Mod 12 : ATT_NightBloomingViolets_animlist
INFO : FNIS Mod 13 : ATT_LovemakingCompendium_animlist
INFO : FNIS Mod 14 : fnis_SeranaHoodFixWithAnim_list
INFO : FNIS Mod 15 : ATT_OAnims_animlist
INFO : FNIS Mod 16 : FNIS_XPMSE_List

### Provide Visual Proof (optional)

![20240902211706_1](https://github.com/user-attachments/assets/f934b194-fe31-4dd4-89f2-d5be31f141bd)

### Contributing To Development

- [X] I read this project's [Contributing To Development](https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/wiki/Contributing-To-Development)

**Comment by WylerL** [comment]
So I managed to put a bandaid on the problem by installing different Vampire Lord animations to overwrite the ones that aren't working. I used Humanoid Vampire Lords, which is a mod that I'm not a huge fan of but it isn't that big of a deal because I don't plan on using vampire lord form at all. I still wouldn't mind getting some advice on this issue though because it has been happening to me repeatedly whenever I uninstall anything creature related, and the only solution I've found is to just run a bunch of other behavior generators(Nemesis, FNIS etc.), and then delete Pandora's cache and run Pandora again.
